src/app/services/webhooks.service.ts

```diff
@@ -158,33 +175,41 @@ const getConsoleMessage = (title, params) => {
 }
 
 // Formats webhook success info into an object to update Submissions collection
-const getSuccessDbUpdate = (response, { webhookUrl, signature }) => {
-  return { webhookUrl, signature, response: getFormattedResponse(response) }
+const getSuccessDbUpdate = (
+  response: AxiosResponse,
+  { webhookUrl, signature }: WebhookParams,
+): IWebhookResponse => {
+  return { webhookUrl, signature, ...getFormattedResponse(response) }
```

Comment:
I feel it's better to leave the key as `response` here instead of destructuring.

This is webhook though, be careful when changing/modifying the returned shapes

Reply:
I changed the output of `getFormattedResponse` so that I could easily type the output as `Pick<IWebhookResponse, 'response'>`. This changed as a result.